CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Fixes
 
+- Add method to download device logs from BrowserStack` [#344](https://github.com/bugsnag/maze-runner/pull/344)
+
+# 6.9.7 - 2022/04/01
+
+## Fixes
+
```

Comment:
This has got a little confused, as 6.9.7 hasn't been released - your PR can go in with it.  I'd rejig to make the whole release as follows (removing the TDB above):
```
# 6.10.0 - 2022/04/01

## Enhancements

- Add method to download device logs from BrowserStack` [#344](https://github.com/bugsnag/maze-runner/pull/344)

## Fixes

- Correct BrowserStack capability to `disableAnimations` [#323](https://github.com/bugsnag/maze-runner/pull/323)

```